src/components/ProfilePage.jsx

```diff
@@ -67,17 +67,30 @@ export class ProfilePage extends React.Component {
     this.props.saveProfile(formId);
   }
 
-  handleChange(formId, name, value) {
-    if (name === 'visibility') {
-      this.props.updateVisibilityDraft(formId, value);
-    } else {
-      this.props.updateAccountDraft(formId, value);
-    }
+  handleChange(name, value) {
+    this.props.updateDraft(name, value);
```

Comment:
Only one kind of draft to update now - everything goes to the same place.

Reply:
party!